torch/_dynamo/bytecode_analysis.py

```diff
@@ -175,7 +175,6 @@ def stacksize_analysis(instructions):
     if False:
         for inst in instructions:
             stack_size = stack_sizes[inst]
-            print(stack_size.low, stack_size.high, inst)
```

Comment:
remove a stray print. Should this be made a log.debug?

Reply:
The entire block `if False:` doesn't run, leading me to believe it's debug code. It's been helpful for me when debugging, so I would hesitate on removing it.

Reply:
Let's either leave this in, or remove the entire `if False:` block.  The entire loop becomes pointless without the print.

Reply:
I'll clean it up the formatting and put the predicate in the config so it's more visible. (Another PR so we don't clobber)